_config.yml

```diff
@@ -17,8 +17,8 @@ title: Intro to Java
 description: This is the homepage of the "Intro to Java" course at ReDI School
 footer_content: "Made with ❤ by teachers at <a href=\"https://www.redi-school.org\">ReDI School</a>."
 aux_links:
-  "Code editor at repl.it":
-    - "https://repl.it/"
+  "Code editor at codebranch.io":
```

Comment:
What's codebranch? Do you mean codeboard.io?

Reply:
I always say code branch for some reason 🙈